test/basic.test.ts

```diff
@@ -0,0 +1,76 @@
+import test from "ava";
+import { rollup } from "rollup";
+import * as path from "path";
+
+import sass from "../src/index";
+import {
+  TEST_UTILS,
```

Comment:
So for TEST_UTILS, could you change the exports to just the methods themselves - Since they already live at 'test/utils' it becomes redundant to export the object `TEST_UTILS` - What would be good here would be to rename `squash` to `stripNewLines`, or something similar!

Reply:
Also, `getOutputFirstChunkCode` could just be `getFirstChunkCode` - since the declaration would read `getFirstChunkCode(output: ...) ...` - Would also change `output` to `chunks`, and/or, `outputChunks` for more correlation between what `output` constants actually represent.